compiler/llvm/clangUtil.cpp

```diff
@@ -2184,10 +2184,34 @@ void runClang(const char* just_parse_filename) {
   clangOtherArgs.push_back("sys_basic.h");
 
   if (!just_parse_filename) {
+
+    if (localeUsesGPU()) {
```

Comment:
I don't have a strong opinion on this matter but I think it'd be reasonable to run this code even for non-GPU compiles. It's just shuffling how we do the includes.